src/lib/Behat/PageElement/ElementFactory.php

```diff
@@ -99,8 +101,21 @@ public static function createElement(UtilityContext $context, string $elementNam
                 return new LanguagePicker($context);
             case UniversalDiscoveryWidget::ELEMENT_NAME:
                 return new UniversalDiscoveryWidget($context);
-            case OldSendForReviewForm::ELEMENT_NAME:
-                return new OldSendForReviewForm($context);
+            case PageBuilderCreatorPopup::ELEMENT_NAME:
+                return new PageBuilderCreatorPopup($context);
+            case 'Content List':
```

Comment:
This will be refactored once Block configuration is available - I'll create dedicated Block classes.

Also it will be moved out of AdminUI to PageBuilder, but it's the next PR.